### PR TITLE
Fixed page titles to show “Kitty Cash” instead of “Skycoin”

### PR DESCRIPTION
**Pull request by bradkendall**
Removed required propTypes on Explorer header to stop errors being thrown
Removed react-intl from Kitties page. (There were no message descriptors for the cats)
Added unique keys to Kitties generated from the loop
Added 3 Kitties to the home page
Removed arbitrary row prop from home page
Fixed overflow CSS issue on homepage
Added ability to show blog posts on the home page

### CODE REVIEW

**Comment by bradkendall** [comment]
screencapture-kittycash-localdev-8080-1519265907456

**Comment by 128-95** [comment]
Looks awesome! Just the font of the summary of the blogposts does not look like a font we are using.

File: src/components/Home/components/HomeKitties/HomeKitties.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+`;
+
+const ButtonContainer = styled.div`
+  padding-bottom: 20px;
+`;
+
+const HomeKitties = () => (
+  <Wrapper>
+    <Container>
+    <KittyTitle>Explore our legendary kitties</KittyTitle>
```
Comment (therealssj) [inline src/components/Home/components/HomeKitties/HomeKitties.js]: `Explore Our Legendary Kitties`?

**Comment by therealssj** [comment]
The black is too overwhelming. I like `#394049` much better as used in `What is Kitty Cash?`

File: src/components/Home/components/HomeKitties/HomeKitties.js (hunk 1 above)
Comment (128-95) [reply src/components/Home/components/HomeKitties/HomeKitties.js]: It are `Legendary Cats` actually.

**Comment by evanlinjin** [comment]
@128-95 So after fixing the fonts it's ready for merge?

**Comment by bradkendall** [comment]
Yes, I will take care of this before I check out tonight. I’m trying to get
the first build of the marketplace out at the moment.

**Comment by bradkendall** [comment]
screencapture-kittycash-localdev-8080-1519369391058

**Comment by bradkendall** [comment]
screencapture-kittycash-localdev-8080-1519369792697

**Pull request merged**